src/components/form/FormRenderer.res

```diff
@@ -655,6 +655,7 @@ module SubmitButton = {
     ~customPaddingClass=?,
     ~textStyle=?,
     ~textWeight=?,
+    ~isLoading=false,
```

Comment:
do we need this extra props to determine the loading state? can't we determine with existing props?

Reply:
it has this 
`let {hasValidationErrors, hasSubmitErrors, submitting, dirtySinceLastSubmit, errors} = formState`
`submitting` hooks but its not behaving as expected so had to add new prop

Reply:
This because on onSubmit function if there is any async function submit hook won't wait for the async function to complete.